corehq/apps/app_manager/dbaccessors.py

```diff
@@ -508,6 +510,23 @@ def _get_save_to_case_updates(domain):
     return save_to_case_updates
 
 
+def _get_case_types_from_apps_query(domain, is_build=False, app_id=None):
+    case_types_agg = NestedAggregation('modules', 'modules').aggregation(
+        TermsAggregation('case_types', 'modules.case_type.exact'))
+    q = (
+        AppES()
+        .domain(domain)
+        .is_build(is_build)
+        .size(0)
+        .aggregation(case_types_agg)
+    )
+    if app_id:
+        q = q.filter(
+            filters.AND(app_id_filter(app_id))
+        )
```

Comment:
nit: this is equivalent to:
```suggestion
        q = q.app_id(app_id)
```
the `AND` filter joins together multiple filters, but only one is passed in here, so it can just be applied directly.